logo/README.md

```diff
@@ -121,3 +123,12 @@ keeps being heated after the print (eg. no `M140 S0` command at the end of the p
 you can change filament and start with the next file. Printing brims can be problematic
 if the printed part only touches the build plate on smaller areas than the rest of the
 body does.
+
+### Importing into Ultimaker Cura
+
+To print in Ultimkaer Cura with two colors, follw these steps:
+* Click on the folder in the upper left corner and choose the first file. Make sure that on the left side extruder 1 is selected.
+* Click again on the folder and choose the second file. Now select extruder 2 on the left side. Both files should now be visible next to each other.
```

Comment:
It's also possible to import both and select the extruder for each part afterwards.